webui/src/pages/user/user-settings-namespaces.tsx

```diff
@@ -0,0 +1,337 @@
+                    label={namespace.name}
+                />;
+            })
+        }
+    </Tabs>;
+};
+
+class UserSettingsNamespacesComponent extends React.Component<UserSettingsNamespacesComponent.Props, UserSettingsNamespacesComponent.State> {
+
+    constructor(props: UserSettingsNamespacesComponent.Props) {
+        super(props);
+
+        this.state = {
+            namespaces: [],
+            members: [],
+            addDialogIsOpen: false,
+            showUserPopper: false,
+            foundUsers: [],
+            popperTarget: undefined,
+            chosenNamespace: undefined
+        };
+    }
+
+    componentDidMount() {
+        this.initNamespaces();
+    }
+
+    render() {
+        return <React.Fragment>
+            {
+                this.state.namespaces.length > 0 && this.state.chosenNamespace ?
+                    <React.Fragment>
+                        <Box className={this.props.classes.namespaceManagementContainer}>
+                            <NamespacesTabs
+                                chosenNamespace={this.state.chosenNamespace}
+                                namespaces={this.state.namespaces}
+                                onChange={this.handleChangeNamespace}
+                            />
+                            <Box className={this.props.classes.memberContainer}>
+                                <Box className={this.props.classes.memberListHeader}>
+                                    <Typography variant='h5'>Members in {this.state.chosenNamespace.name}</Typography>
+                                    <Button className={this.props.classes.addButton} variant='outlined' onClick={this.handleOpenAddDialog}>
+                                        Add Namespace Member
+                                    </Button>
+                                </Box>
+                                <Paper>
+                                    {this.state.members.map(member =>
+                                        <UserNamespaceMember
+                                            key={'nspcmbr-' + member.user.loginName + member.user.provider}
+                                            member={member}
+                                            user={this.props.user}
+                                            service={this.props.service}
+                                            onRemoveUser={() => this.removeUser(member)} />)}
+                                </Paper>
+                            </Box>
+                        </Box>
+                        {this.renderAddDialog()}
+                    </React.Fragment>
+                    : <div>No Namespaces Available. 
+                        Read <Link color='secondary' href='https://github.com/eclipse/openvsx/wiki/Publishing-Extensions' target='_blank'>here</Link> about creating namespaces.
+                    </div>
+            }
+        </React.Fragment>;
+    }
+
+    protected renderAddDialog() {
```

Comment:
Currently it's not possible to close the dialog. Close it with `esc` key or by clicking outside of the dialog. `enter` key should apply the selected user, even if its name was not fully entered (only if the search resulted in exactly one user).